server/app/forms/translation/ProgramTranslationForm.java

```diff
@@ -163,6 +184,9 @@ private static ImmutableList<String> allFieldNames(
       builder.add(
           statusKeyToUpdateFieldName(i), localizedStatusFieldName(i), localizedEmailFieldName(i));
     }
+    for (int i = 0; i < applicationSteps.size(); i++) {
```

Comment:
With all of these changes I mostly followed the pattern setup by screen name/description updates.